lab_4_retrieval_w_clustering/main.py

```diff
@@ -81,6 +99,16 @@ def _calculate_bm25(self, tokenized_document: list[str]) -> Vector:
         Returns:
             Vector: BM25 vector for document.
         """
+        if not tokenized_document or not isinstance(tokenized_document, list):
+            raise ValueError
+        vector_list = [0.0 * n for n in range(len(self._vocabulary))]
```

Comment:
`0.0 * n` -> `0.0` I believe you wanted another thing

Reply:
but it works fine - it creates a list with n number of zeros

Reply:
Of course, remove `* n`

Reply:
done?..